src/acceleration/BaseQNAcceleration.hpp

```diff
@@ -203,6 +204,9 @@ class BaseQNAcceleration : public Acceleration {
   /// @brief Stores x tilde deltas, where x tilde are values computed by solvers.
   Eigen::MatrixXd _matrixW;
 
+  /// @brief  Toggle to switch between using all of the substeps when constructing V_k or only the last time step of the time window.
+  bool _reduced = true;
```

Comment:
```suggestion
  /// @brief Toggle to switch between using all of the substeps when constructing V_k or only the last time step of the time window.
  bool _reduced = true;
```
Are you referring to https://onlinelibrary.wiley.com/doi/epdf/10.1002/nme.6443 somewhere? I think otherwise the term `_reduced` might sound a bit cryptic. You could even consider writing " if `_reduced = false` uses the full QN-WI and if `_reduced = true` uses rQN-WI form paper ..."